src/_gettsim/interface.py

```diff
@@ -604,22 +604,23 @@ def _add_rounding_to_functions(functions, params):
             functions_new[func_name] = _add_rounding_to_one_function(
                 base=rounding_spec["base"],
                 direction=rounding_spec["direction"],
+                add=rounding_spec["add"] if "add" in rounding_spec else None,
             )(func)
 
     return functions_new
 
 
-def _add_rounding_to_one_function(base, direction):
+def _add_rounding_to_one_function(base, direction, add):
```

Comment:
Do we have tests for this or is codecov's complaint a fluke?

And please add typing, something like

```python
base: float,
direction: Literal['up', 'down', 'nearest'']
add: float
```

Not sure about what you mean by "optional"? Surely, the function expects to be called with `add` supplied. Do you mean that the typing should be `add: float|None` ?

However, given all the checks below, I'd think it'll be easier to just specify a 0 everywhere.

And we should be a bit more expressive in the name, maybe `to_add_after_rounding` ?

Reply:
> Do we have tests for this or is codecov's complaint a fluke?

If you are referring to the codecov report from above: this was a mistake on the codecov side. If not, I don't know what you are referring to.

> However, given all the checks below, I'd think it'll be easier to just specify a 0 everywhere.

I agree, I changed the default value from `None` to `0`:

```python
to_add_after_rounding=rounding_spec["to_add_after_rounding"]
                if "to_add_after_rounding" in rounding_spec
                else 0,
            )(func)
``